opentelemetry-proto/README.rst

```diff
@@ -0,0 +1,19 @@
+OpenTelemetry Python Proto
+==========================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-proto.svg
+   :target: https://pypi.org/project/opentelemetry-proto/
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-proto
+
+References
+----------
+
+* `OpenTelemetry Project <https://opentelemetry.io/>`_
```

Comment:
Please add a link to the opentelemetry-proto repo. Would be nice to also describe how the protos were generated in the readme somewhere.

Reply:
Fixed